packages/unity-react-core/src/core/utils/index.js

```diff
@@ -131,6 +132,12 @@ export const initVideo = ({ targetSelector, props }) =>
 export const initTabbedPanels = ({ targetSelector, props }) =>
   RenderReact(TabbedPanels, props, document.querySelector(targetSelector));
 
+/**
+  * @param {ComponentProps} props
+  */
+export const initDivider = ({ targetSelector, props }) =>
```

Comment:
I was going to say, Let's keep up the practice of putting these in alpha order... but I see we've already departed somewhere along the line. :P Not required. Consider it 5 bonus points. Get 1000 bonus points and you can redeem it for a stick of Juicy Fruit.